### ISSUE TITLE
Using Steam account on Windows makes Steam issues on Linux..

### ISSUE THREAD

**Issue by Zorrototo** [ValveSoftware/steam-for-linux#6537]
#### Your system information:

* Steam client version (build number or date): steam-runtime_0.20190711.3
* Distribution (e.g. Ubuntu): Manjaro KDE
* Opted into Steam client beta?: [No]
* Have you checked for system updates?: [Yes]
* [system information](https://gist.github.com/Zorrototo/533bb4f3b9d8d5abc137cd8cf5c00f02)

#### Please describe your issue in as much detail as possible:

After using my Steam account on Windows, when I reboot to my Linux disk and start Steam, Steam thinks all or most of all my Proton games require an update. If I force all these updates manually they instantly end (0 bytes downloads), but the real issue here is that it will re-download most/all the redistributable (Common Steamworks things) for my Proton games, without any other reason than I rebooted previously on Windows and logged into my account. This is always the case when I reboot on Windows and start Steam, then reboot again on Linux this always happen.

#### Steps for reproducing this issue:

1. Reboot on Windows, start Steam, and play some games.
2. Close Steam. Stop computer.
3. Reboot on Linux, start Steam, check you Download in Steam and see many games seems to require an update (0 bytes updates), and it downloads all Redistributable for the installed Proton games.

**Comment by kisak-valve** [comment]
Hello @Zorrototo, this issue is already being tracked at #6064. Closing as a duplicate.

**Comment by Zorrototo** [comment]
The problem is not the 0 bytes update, but THE RE-DISTRIBUTABLE (directX, vcredist, and so on...) of my proton game downloading every time.

I had the issue with the shader pre caching downloading again every time I was starting Steam, but I disable shader pre caching, and it solved this issue, this is another issue.

If I don't boot on Windows and start my Steam (like if I only reboot on Linux), I don't have to download redistributable every time I start Steam, it is only after booting and logging into Steam on Windows then going back to Linux that this issue occurs.

**Comment by kisak-valve** [comment]
Are you using the same Steam library with both OSes?

If that's the case, https://github.com/ValveSoftware/steam-for-linux/blob/master/RelNotes.md#installation notes that this is not supported and can lead to unexpected quirks.

**Comment by Zorrototo** [comment]
No, This is what I tried first time I installed Linux and noticed the problem back then. I have everything separated, my Linux SSD only has Linux Steam and library, and my Windows Steam is on the Windows system disk alone. Nothing in common between the two systems.
I don't even mount my Windows SSD on Linux.

**Comment by Zorrototo** [comment]
Today I played on my Windows system, then after rebooting to Windows, Steam didn't download anything on the Linux Steam. Did something got fixed?
